### PR TITLE
feat: Pagination in Attendees list

### LINKED ISSUE #1279: Attendees not loading automatically
### Bug Report    

**Actual Behaviour**

1. Attendees are not loading automatically for any event: 
2. Timeout occurs every time for FOSSASIA Summit 2018 as it contains huge number of attendees - attendees can't be loaded even after refresh.

**Expected Behaviour**

Should load automatically when Check In is selected

**Steps to reproduce it**

Select `Check In` from drawer

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1279 

**Checklist:**

- [ ] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream development branch.

**Changes:** 

Implemented pagination in Attendees list.

**GIF for the change:**

**Earlier:**
Results in timeout since this is an event with a huge number of attendees.

ezgif com-video-to-gif (70)

**Now:**
ezgif com-optimize (3)

### CODE REVIEW

**Comment by ShridharGoel** [comment]
@iamareebjamal This is complete and working. Please review.

File: app/src/main/java/com/eventyay/organizer/data/attendee/AttendeeApi.java [hunk 1]
```diff
@@ -7,12 +7,16 @@
...
 import retrofit2.http.PATCH;
 import retrofit2.http.Path;
+import retrofit2.http.Query;
 
 public interface AttendeeApi {
 
     @GET("events/{id}/attendees?include=order,ticket,event&fields[event]=id&fields[ticket]=id&page[size]=0")
     Observable<List<Attendee>> getAttendees(@Path("id") long id);
 
+    @GET("events/{id}/attendees?include=order,ticket,event&fields[event]=id&fields[ticket]=id&page[size]=50")
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/data/attendee/AttendeeApi.java]: 50 is still quite large. Use 20

**Comment by iamareebjamal** [comment]
Use proper camel case for variable names and function names

**Comment by ShridharGoel** [comment]
> Use proper camel case for variable names and function names

You mean 'PageWise' instead of 'Pagewise' ? Anything else also?

File: app/src/main/java/com/eventyay/organizer/data/attendee/AttendeeApi.java (hunk 1 above)
Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/data/attendee/AttendeeApi.java]: Okay

**Comment by iamareebjamal** [comment]
variable and parameter names as well

**Comment by ShridharGoel** [comment]
@iamareebjamal I have used FIRST_PAGE as the variable name because it's a constant. Does it need to be changed?

**Comment by ShridharGoel** [comment]
Others are fixed

**Comment by ShridharGoel** [comment]
@iamareebjamal Please review.

File: app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java [hunk 2]
```diff
@@ -74,6 +79,11 @@
     private SwipeRefreshLayout refreshLayout;
     private SearchView searchView;
 
+    private static final long FIRST_PAGE = 1;
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java]: Reuse from presenter

**Pull request merged**